pkg/client/userd/daemon/grpc.go

```diff
@@ -155,6 +155,14 @@ func (s *Service) Status(c context.Context, ex *empty.Empty) (result *rpc.Connec
 			})
 		} else {
 			result = s.session.Status(s.sessionContext)
+			err = s.WithSession(c, "status", func(ctx context.Context, session userd.Session) error {
```

Comment:
It is redundant to call `WithSession` here, given that the sessionLock is in place already.